plugins/org.jboss.tools.openshift.ui/src/org/jboss/tools/openshift/internal/ui/wizard/deployimage/DeployImageWizardModel.java

```diff
@@ -93,6 +95,18 @@
 	
 	private final List<String> imageNames = new ArrayList<>();
 	
+    private List<IDockerConnection> dockerConnections = Arrays.asList(DockerConnectionManager.getInstance().getConnections());
+
+	
+	public DeployImageWizardModel() {
+	    DockerConnectionManager.getInstance().addConnectionManagerListener(this);
+	}
+	
+	@Override
```

Comment:
This doesn't compile for me, when I rebase against master. Which interface declares release()?